refactoring exercises (#789)
devopsacademyau/academy: classes/02class/exercises/c02-network02/README.md

# Subnetting

> ### Given a AWS network range, slice it in 4 small subnets and identify how many usable hosts for each one of them.

Reviewer: ### 
```suggestion
### Given an AWS network range, slice it in 4 small subnets and identify the number of usable hosts(IPs) for each one of them.
```
